translator/translate/otel/processor/awsappsignals/testdata/invalidRulesConfig.json

```diff
@@ -6,7 +6,8 @@
         "limiter": {
           "drop_threshold": 20,
           "log_dropped_metrics": true,
-          "rotation_interval": "10m"
+          "rotation_interval": "10m",
+          "garbage_collection_interval": "10m"
```

Comment:
Was this added to the schema.json?

Reply:
yeah, also not sure if this was meant to be configurable by customers

Reply:
Ppl can use it even if its not in our schema afaik depending on additonalPros..
None of the metric limiter props are in our schema.json - we should revisit.
For now, I just added to all the places `rotation_interval` is present.